setup.py

```diff
@@ -22,13 +22,12 @@
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.2',
```

Comment:
While you're at it, you can also remove 3.2 which isn't supported either. I'm not doing this to avoid a conflict for you.

Thanks,